### ISSUE TITLE
git revision for NorwegianCourtsBitextMining

### ISSUE THREAD

**Issue by avidale** [embeddings-benchmark/mteb#220]
When I am running the task NorwegianCourtsBitextMining, I get an error downloading the dataset:
```
datasets.exceptions.DatasetNotFoundError: Dataset 'kardosdrur/norwegian-courts' doesn't exist on the Hub at revision '3bc5cfb4ec514264fe2db5615fac9016f7251552'
```

When we take a look at the dataset repo history (https://huggingface.co/datasets/kardosdrur/norwegian-courts/commits/main), the commit with such hash does not exist indeed. 

Which revision should one use?

**Comment by avidale** [comment]
Tagging @x-tabdeveloping as the task author

**Comment by x-tabdeveloping** [comment]
Hey @avidale sorry for the troubles, and thanks for reminding me to fix this issue. In the [Scandinavian Embedding Benchmark](https://kennethenevoldsen.github.io/scandinavian-embedding-benchmark/), I set the revision to `None` as I suspect that the dataset is highly unlikely to change.
I will make sure to submit a pull request today addressing the problem.

**Comment by KennethEnevoldsen** [comment]
Actually @x-tabdeveloping I have updated it to a specific commit on [Scandinavian Embedding Benchmark](https://kennethenevoldsen.github.io/scandinavian-embedding-benchmark/) so that even if it changes we would have to change it. We will add a PR to MTEB based on the SEB benchmark sometime in the future.

**Comment by x-tabdeveloping** [comment]
The PR already exists, I can change it in a commit.

**Comment by KennethEnevoldsen** [comment]
Merged in #221 will close this issue.
